master/getting-started/bare-metal/policy/objects.md

```diff
@@ -97,14 +97,24 @@ and (if used) the interface name in the configuration. If there was
 something wrong with the endpoint data, Felix will log a validation
 error at `WARNING` level and it will ignore the endpoint:
 
-    $ grep "Validation failed" /var/log/calico/felix.log
-    2016-05-31 12:16:21,651 [WARNING][8657/3] calico.felix.fetcd 1017:
-        Validation failed for host endpoint HostEndpointId<eth0>, treating as
-        missing: 'name' or 'expected_ipvX_addrs' must be present.;
-        '{ "labels": {"foo": "bar"}, "profile_ids": ["prof1"]}'
+A `grep` through the Felix logs for the string "Validation failed" should allow
```

Comment:
I think this whole block can probably be removed now - I think we do validation in libcalico now, so we won't allow a user to write an invalid host endpoint to the datastore.

@fasaxc was this text meant to cover something else?